README.md

```diff
@@ -1,22 +1,22 @@
-![Tests](https://github.com/oamg/rhc-worker-bash/actions/workflows/tests.yml/badge.svg)
-[![codecov](https://codecov.io/github/oamg/rhc-worker-bash/branch/main/graph/badge.svg?token=6MRLOJS2SJ)](https://codecov.io/github/oamg/rhc-worker-bash)
+![Tests](https://github.com/oamg/rhc-worker/actions/workflows/tests.yml/badge.svg)
+[![codecov](https://codecov.io/github/oamg/rhc-worker/branch/main/graph/badge.svg?token=6MRLOJS2SJ)](https://codecov.io/github/oamg/rhc-worker)
 
 # RHC Worker Bash
 
 Remote Host Configuration (rhc) worker for executing bash scripts on hosts
 managed by Red Hat Insights.
```

Comment:
Maybe something like this? Also depending on the title change to outline below needed

```suggestion
# RHC Worker

Remote Host Configuration (rhc) worker for executing  scripts on hosts
managed by Red Hat Insights. Interpreter used to execute the script is defined inside the supplied yaml file - served by insights.
```